extension-base/ctp-extension-azure-adapter.js

```diff
@@ -0,0 +1,39 @@
+// Azure
+
+module.exports = {
+  ctpExtensionAdapter: function(fn) {
+    return function(context, req) {
+      fn(req.body, ctpResponse(context), context.log);
+    }
+  }
+}
+
+function ctpResponse(context) {
+  return {
+    reject: createErrors(context),
```

Comment:
Isnt this createReject?

Reply:
The compiler didn't complain after my refactoring! Oh, wait... 😨